provider/cpu_utils.py

```diff
@@ -99,3 +101,32 @@ def check_guest_cpu_topology(session, os_type, cpuinfo):
         logging.debug("CPU infomation of guest:\n%s", out)
 
     return is_matched
+
+
+def check_flags(params, flags, test, session=None):
```

Comment:
How about rename it to check_cpu_flags? Others look good to me.

Reply:
> How about rename it to check_cpu_flags? Others look good to me.

@huangyum  update, thanks.

Reply:
Would you pls provide the latest test results? Thanks.

Reply:
> Would you pls provide the latest test results? Thanks.

@huangyum update, please help to check, thanks.
 Host_RHEL.m8.u3.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.x86_cpu_flags.intel.memory_protection_key.q35: PASS (95.71 s)
 Host_RHEL.m8.u3.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.x86_cpu_model.Skylake-Client-noTSX.q35: PASS (58.88 s)